### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by onurravli**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1041,3 +1041,21 @@ XML solves this problem by being readable to neither.
 **A:** Python.
 
 ---
+
+**Q:** How many programmers does it take to change a lightbulb?
+
+**A:** None, that's a hardware problem.
```
Comment (styfle) [inline readme.md:1047]: Duplicate

image

File: readme.md [hunk 2]
```diff
@@ -1041,3 +1041,21 @@ XML solves this problem by being readable to neither.
...
+
+---
+
+**Q:** Why did the programmer quit his job?
+
+**A:** Because he didn't get arrays.
```
Comment (styfle) [inline readme.md:1053]: Duplicate 


image

File: readme.md [hunk 3]
```diff
@@ -1041,3 +1041,21 @@ XML solves this problem by being readable to neither.
...
+
+---
+
+**Q:** Why do programmers prefer dark mode?
+
+**A:** Because light attracts bugs.
```
Comment (styfle) [inline readme.md:1059]: Duplicate

image

**Review by styfle — CHANGES_REQUESTED** [review]
These are all duplicates

**Comment by onurravli** [comment]
I should have check. Sorry.

**Pull request closed without merging**